rotkehlchen/data_import/importers/cointracking.py

```diff
@@ -167,24 +168,18 @@ def _consume_cointracking_entry(
                 asset = asset_resolver(csv_row['Cur.Sell'])
                 movement_type = HistoryEventType.WITHDRAWAL
 
-            events = [AssetMovement(
-                location=location,
-                event_type=movement_type,
-                timestamp=ts_sec_to_ms(timestamp),
-                asset=asset,
-                amount=amount,
-            )]
-            if fee != ZERO:
-                events.append(AssetMovement(
-                    event_identifier=events[0].event_identifier,
+            self.add_history_events(
+                write_cursor=write_cursor,
+                history_events=create_asset_movement_with_fee(
+                    timestamp=ts_sec_to_ms(timestamp),
```

Comment:
you seem to always use the timestamp in ms in this function so you can have it converted when deserialized and avoid calling the function ts_sec_to_ms in each one